### DISCUSSION TITLE
[Vote ended on 2023-03-27] Use a forum to build project-wide participation & discussion

### DISCUSSION THREAD

**Discussion by GregSutcliffe** [ansible-community/community-topics#211] [Votes done]
Proposal & Discussion: #202

Discussion should be kept to that issue, please.

This vote is to:
- Use a forum to build project-wide participation, discussion,and support
- Commission the Red Hat Ansible Community Team to build the forum
 
That 2nd point is because our team have the budget, but we want the community's approval to be explicit - this is a big project :)

Please vote with +1 to do this and -1 to keep things as they are (in all their many many places).

**Comment by pidreher** [comment]
+1

**Comment by apple4ever** [comment]
+1

**Comment by chriscroome** [comment]
+1

**Comment by felixfontein** [comment]
+1

**Comment by cybette** [comment]
+1

**Comment by chofstede** [comment]
+1

**Comment by Andersson007** [comment]
+1

**Comment by ssbarnea** [comment]
-1 As we already have a forum, github discussions is a forum, one with unbeatable integration with our projects and that requires no budget or maintenance.

**Comment by gundalow** [comment]
+1

**Comment by cidrblock** [comment]
+1

**Comment by briantist** [comment]
+1

**Comment by samccann** [comment]
+1

**Comment by RaggedStaff** [comment]
+1

**Comment by maxamillion** [comment]
+1

**Comment by leogallego** [comment]
+1

**Comment by mariolenz** [comment]
+1

**Comment by alinabuzachis** [comment]
+1

**Comment by tremble** [comment]
+1

**Comment by markuman** [comment]
-1

**Comment by shanemcd** [comment]
+1

**Comment by gotmax23** [comment]
+1

**Comment by russoz** [comment]
+1

**Comment by schurzi** [comment]
+1
